packages/synapse-react-client/src/components/OAuthClientManagement/CreateOAuthClient.tsx

```diff
@@ -106,31 +101,37 @@ export const CreateOAuthModal: React.FunctionComponent<
   useEffect(() => {
     setClientName(client?.client_name ?? '')
     setRedirectUris(
-      client?.redirect_uris.map(str => ({ uri: str })) ?? [{ uri: '' }],
+      client?.redirect_uris?.map((str: string) => ({ uri: str })) ?? [
+        { uri: '' },
+      ],
```

Comment:
Note that redirect_uris is NOT required in the spec, so we have to do additional null handling or assertions to work with the new model. This will be common, and is IMO the biggest downside and reason to not use the generated code right now.